luda/editor/client/src/saver/saver.ts

```diff
@@ -3,11 +3,11 @@ import { ISaver } from "./ISaver";
 
 class Saver implements ISaver {
   private lastValueString?: string;
-  private isSaving_: boolean = false;
-  private isUpToDate_: boolean = false;
```

Comment:
I know why you do this. Unlike C#, typescript don't have simple way to make private-set-public-get property.
I think saver is just the implementation of ISaver, so I think saver can expose `isSaving` and `isUpToDate` if we wrap saver with ISaver. That's why i change this code like it.